source/mir/las/sum.d

```diff
@@ -1988,6 +2065,28 @@ unittest
         assert(s3.array == c.array);
     }
 }
+unittest
+{
+    import core.simd;
+    import std.range: iota;
+    import std.array: array;
+
+    double[] ns = [9.0, 101.0];
```

Comment:
there are quite many specializations for `pairwise` - here we add tests for the non-tested parts.